robottelo/ui/session.py

```diff
@@ -28,12 +28,13 @@ def __enter__(self):
         return self
 
     def __exit__(self, exc_type, exc_value, traceback):
-        self.logout()
+        #self.logout()
+        pass
```

Comment:
Need to confirm, but the `exc_type` should contain a instance of `AssertionError` if a test fail. I think we can take advantage of it and only logout if there is no `AssertionError`.